scripts/ksqlDB/statements.sql

```diff
@@ -1,6 +1,6 @@
 CREATE STREAM wikipedia WITH (kafka_topic='wikipedia.parsed', value_format='AVRO');
-CREATE STREAM wikipedianobot AS SELECT * FROM wikipedia WHERE isbot <> true;
-CREATE STREAM wikipediabot AS SELECT * FROM wikipedia WHERE isbot = true;
-CREATE TABLE en_wikipedia_gt_1 AS SELECT username, wikipage, count(*) AS COUNT FROM wikipedia WINDOW TUMBLING (size 300 second) WHERE channel = '#en.wikipedia' GROUP BY username, wikipage HAVING count(*) > 1;
-CREATE STREAM en_wikipedia_gt_1_stream (USERNAME string, WIKIPAGE string, COUNT bigint) WITH (kafka_topic='EN_WIKIPEDIA_GT_1', value_format='AVRO');
+CREATE STREAM wikipedianobot AS SELECT *, (length->new - length->old) AS BYTECHANGE FROM wikipedia WHERE bot = false;
+CREATE STREAM wikipediabot AS SELECT *, (length->new - length->old) AS BYTECHANGE FROM wikipedia WHERE bot = true;
+CREATE TABLE en_wikipedia_gt_1 AS SELECT user, 'meta.uri' AS URI, count(*) AS COUNT FROM wikipedia WINDOW TUMBLING (size 300 second) WHERE 'meta.domain' = 'commons.wikimedia.org' GROUP BY user, 'meta.uri' HAVING count(*) > 1;
+CREATE STREAM en_wikipedia_gt_1_stream (USER string, URI string, COUNT bigint) WITH (kafka_topic='EN_WIKIPEDIA_GT_1', value_format='AVRO');
```

Comment:
When running this demo, I think I'm hitting permissions issues with the aggregations being done in this stream. The underlying topics never populate with data. 
I say "permissions issues" because when trying to run the `SELECT` portion of this query via the ksqlDB cli with the user suggested in the demo (ksqlDBUser), I hit this error: ```Not authorized to access topics: [_confluent-ksql-ksql-clustertransient_2541491807929176778_1603830231880-Aggregate-GroupBy-repartition]```
Although, when I run with superUser I don't hit any issues with that query.

Very similar errors are present in the ksqlDB container logs. See below:
```[2020-10-27 19:44:53,784] ERROR [Consumer clientId=_confluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193-d9a0c497-01fd-4263-9e4d-c238a1156ddf-StreamThread-1-consumer, groupId=_con\
fluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193] Topic authorization failed for topics [_confluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193-Aggregate-GroupBy\
-repartition] (org.apache.kafka.clients.Metadata)
[2020-10-27 19:44:53,784] INFO [Consumer clientId=_confluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193-d9a0c497-01fd-4263-9e4d-c238a1156ddf-StreamThread-1-consumer, groupId=_conf\
luent-ksql-ksql-clustertransient_6970312038198470337_1603827890193] Cluster ID: ExN2tQD5SmaxbnyDFTAwiQ (org.apache.kafka.clients.Metadata)
[2020-10-27 19:44:53,785] ERROR stream-thread [_confluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193-d9a0c497-01fd-4263-9e4d-c238a1156ddf-StreamThread-1] Encountered the following\
 exception during processing and the thread is going to shut down:  (org.apache.kafka.streams.processor.internals.StreamThread)
org.apache.kafka.common.errors.TopicAuthorizationException: Not authorized to access topics: [_confluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193-Aggregate-GroupBy-repartition]
[2020-10-27 19:44:53,785] INFO stream-thread [_confluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193-d9a0c497-01fd-4263-9e4d-c238a1156ddf-StreamThread-1] State transition from STAR\
TING to PENDING_SHUTDOWN (org.apache.kafka.streams.processor.internals.StreamThread)
[2020-10-27 19:44:53,785] INFO stream-thread [_confluent-ksql-ksql-clustertransient_6970312038198470337_1603827890193-d9a0c497-01fd-4263-9e4d-c238a1156ddf-StreamThread-1] Shutting down (org.apache.\
kafka.streams.processor.internals.StreamThread)```

Reply:
Note it does look like "ksqlDBUser" does submit these statements, so I would expect the queries to run into similar permissions issues that I ran into when running the `SELECT` statements via a cli session. I suggest either changing the permissions for "ksqlDBUser" or submitting the statements with "superUser" in `scripts/ksqlDB/run_ksqlDB.sh`.

Reply:
@awalther28 for clarification, can you please share the statement that is causing the failure?  I can run `select * from en_wikipedia_gt_1_stream emit changes;` without auth issues (though at the moment the stream looks empty...)

Reply:
I would suspect that query you posted would be fine because it isn't performing an aggregation, which is what I think the permissions issues are referring to. The following is the query + error I was referring to.
`SELECT user, 'meta.uri' AS URI, count(*) AS COUNT FROM wikipedia GROUP BY user, 'meta.uri' EMIT CHANGES LIMIT 3;` 
![Screen Shot 2020-10-28 at 8 11 29 AM](https://user-images.githubusercontent.com/12937975/97447498-3af32700-18f5-11eb-8c34-aa218c71740e.png)

Reply:
Fixed via https://github.com/confluentinc/cp-demo/pull/265/commits/8be16cab0dac24115d2917989d576734e9b764e4